napari/conftest.py

```diff
@@ -15,6 +16,9 @@
 from napari.utils import io
 from napari.utils.config import async_loading
 
+if not hasattr(pooch.utils, 'file_hash'):
+    setattr(pooch.utils, 'file_hash', pooch.hashes.file_hash)
```

Comment:
```suggestion
from pooch import file_hash
```
I looked into this a bit on the scikit-image side and this import has always been available from the top-level namespace.

Corresponding scikit-image fix in: https://github.com/scikit-image/scikit-image/pull/5529
A `pooch` dev confirmed that they had not considered the `utils` submodule public API: https://github.com/scikit-image/scikit-image/issues/5528#issuecomment-903854574

Reply:
thanks @grlee77, unfortunately `from pooch import file_hash` does not work here, since it's the scikit-image import that's causing the issue (not our own import).  So this fix here is just making that name available for scikit-image, in case someone is testing napari with a version of scikit-image prior to scikit-image/scikit-image#5529 but later than pooch 1.5

Reply:
I see, thanks (should have read the linked issue here!). I can try to make a 0.18.3 release ASAP for scikit-image

Reply:
no worries!  thanks for stopping by :)